### DISCUSSION TITLE
Syntax error: Identifier 'getServerSideProps' has already been declared

### DISCUSSION THREAD

**Discussion by tinahmgao** [vercel/next.js#14694] [Help] [answered]
> I use the same syntax in **index.js** and the **movie.js**, both of them are pages.  Is this mean I only can export the getServerSideProps once in index.js?

```javascript
export async function getServerSideProps(context) {
  console.log(context)
  return {
    props: {}, // will be passed to the page component as props
  }
}
```

**Comment by jamesmosier** [comment, marked as answer]
You can only export `getServerSideProps` once per page in `pages/*`. 

Are you by chance importing one file (`page`) into the other?

**Reply by tinahmgao** [reply]
truly it is the problem, seem the text editor automatically adds the import into the page. thank u!

**Comment by PRASAD7719** [comment]
Module parse failed: Identifier 'getServerSideProps' has already been declared (101:22)
|     }, void 0, true);
| }
> export async function getServerSideProps() {
|     try {
|         const res = await fetch(process.env.BASEURL + '/api/aboutus');

Import trace for requested module:  i got this issue and i have also declared only one page this getServerSideProps prop so lets menow what isiue i got
